src/runtime/Types/OperatorMethod.cs

```diff
@@ -54,6 +54,7 @@ static OperatorMethod()
 
                 ["__int__"] = new SlotDefinition("__int__", TypeOffset.nb_int),
                 ["__float__"] = new SlotDefinition("__float__", TypeOffset.nb_float),
+                ["__index__"] = new SlotDefinition("__float__", TypeOffset.nb_index),
```

Comment:
`SlotDefinition("__float__", ...)` is this a mistake?

Reply:
It is, thanks!